MobileFilter/sections/specific_web.txt

```diff
@@ -60,6 +60,7 @@ inmywordz.com##.ad-single-top
 league-funny.com##.bottomFixedBanner
 m.hindustantimes.com##[class="fl100 mt10"][style="height:100px"]
 m.hindustantimes.com###stickydivAD
+m.hindustantimes.com##body > section[style="height:100px"]
```

Comment:
When you are adding a new rules, try to look at the old ones.
For example, you can change a bit this rule `m.hindustantimes.com##[class="fl100 mt10"][style="height:100px"]` to fix the issue

Reply:
I would better remove `m.hindustantimes.com##[class="fl100 mt10"][style="height:100px"]` and leave 
`m.hindustantimes.com##body > section[style="height:100px"]`
They removed mt10 class, they may remove fl100

Reply:
They also can change their HTML Code in the future - nothing lasts forever